app/views/podcast_planner/_form_draft_settings.html.erb

```diff
@@ -34,7 +34,7 @@
   <div class="container">
     <div class="row">
       <% feeds = @podcast.feeds.tab_order %>
-      <% defaults = feeds.filter_map { |f| f.id if f.default? || f.apple? } %>
+      <% defaults = feeds.filter_map { |f| f.id if f.default? || f.integration_type } %>
```

Comment:
Okay, so by default... episodes go to the default feed, and any integration feeds.

Reply:
yes, that's how apple worked already, and it also matches the requirements for the megaphone integration